pkg/kritis/crd/imagesecuritypolicy/imagesecuritypolicy.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package imagesecuritypolicy
+
+import (
+	"fmt"
+	"github.com/grafeas/kritis/pkg/kritis/apis/kritis/v1beta1"
+	clientset "github.com/grafeas/kritis/pkg/kritis/client/clientset/versioned"
+	"github.com/grafeas/kritis/pkg/kritis/constants"
+	"github.com/grafeas/kritis/pkg/kritis/metadata"
+	ca "google.golang.org/genproto/googleapis/devtools/containeranalysis/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type ImageSecurityPolicy struct {
+	v1beta1.ImageSecurityPolicy
```

Comment:
Hmm, it looks like we're embedding the CRD struct directly here. Would it make sense to rename this outer struct to be something else? And maybe hold the vulnz client object?

Reply:
I ended up getting rid of the struct and passing in the CRD to the functions. Decided not to get the pods here because I figured it would make more sense to have that in the library where we also modify pod labels/annotations.